internal/cache/controller_obj_cache.go

```diff
@@ -2605,7 +2605,7 @@ func checkRequiredValuesYaml(returnErr *error) bool {
 
 	lib.SetNamePrefix()
 	// after clusterName validation, set AKO User to be used in created_by fields for Avi Objects
-	lib.SetAKOUser()
+	lib.SetAKOUser("ako-")
```

Comment:
shall we make this as a constant? Two constants - `ako-` from ako pod and `ako-gw-api-` from gateway API pod?